pyproject.toml

```
@@ -145,3 +147,10 @@ max-complexity = 10
     "F401", # unused import
     "F403", # star imports
 ]
+
+[tool.pytest.ini_options]
+addopts = "--strict-markers"
+markers =[
+    "jobs",
+    "unit",
```

```suggestion
    "agent-jobs",
```
Let's hold off on adding additional markers that we aren't using yet.
Also, can we make `jobs` less ambiguous.

---

There are already unit markers in the codebase. When I did not have the unit marker present in that list errors started popping up.